RallyHereIntegration/Source/RallyHereIntegration/Private/RH_SettingsSubsystem.cpp

```diff
@@ -0,0 +1,29 @@
+#include "RH_SettingsSubsystem.h"
+
+URH_SettingsSubsystem::URH_SettingsSubsystem(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+}
+
+void URH_SettingsSubsystem::GetSettingTypes(const FRH_GenericSuccessWithErrorBlock& Delegate)
+{
+	UE_LOG(LogRallyHereIntegration, VeryVerbose, TEXT("[%s]"), ANSI_TO_TCHAR(__FUNCTION__));
+
+	auto Request = TGetConfigSettings::Request();
+	Request.AuthContext = GetAuthContext();
+
+	auto Helper = MakeShared<FRH_SimpleQueryHelper<TGetConfigSettings>>(
+		TGetConfigSettings::Delegate::CreateLambda([this, Delegate](const TGetConfigSettings::Response& Resp)
+			{
+				if (Resp.IsSuccessful())
```

Comment:
Existing check does not hurt, but the first delegate here is only called if Resp.IsSuccessful() is true. Does not get called for semi-succesful non-updates (ex: NOT_MODIFIED codes).

The second delegate (completion delegate) always gets called.